pinot-core/src/main/java/org/apache/pinot/core/transport/InstanceRequestHandler.java

```diff
@@ -184,9 +185,9 @@ private void sendResponse(ChannelHandlerContext ctx, String tableNameWithType, l
       _serverMetrics.addTimedTableValue(tableNameWithType, ServerTimer.NETTY_CONNECTION_SEND_RESPONSE_LATENCY,
           sendResponseLatencyMs, TimeUnit.MILLISECONDS);
 
-      int totalQueryTimeMs = (int) (sendResponseEndTimeMs - queryArrivalTimeMs);
-      if (totalQueryTimeMs > SLOW_QUERY_LATENCY_THRESHOLD_MS) {
-        LOGGER.info(
+      long totalQueryTimeMs = sendResponseEndTimeMs - queryArrivalTimeMs;
+      if (totalQueryTimeMs > _slowQueryLatencyThresholdMs) {
+        LOGGER.warn(
```

Comment:
If we are adding a warning in the `BrokerRequestHandler`, then this warning can be removed.

Reply:
This can still be useful? Since it shows the latency to send the response to broker which is not available in the `INFO` log? Also this can help to identify slow servers?